crates/forge/bin/cmd/test/mod.rs

```diff
@@ -113,8 +113,8 @@ pub struct TestArgs {
 
     /// Max concurrent threads to use.
     /// Default value is the number of available CPUs.
-    #[arg(long)]
-    pub max_threads: Option<u64>,
+    #[arg(long, short, visible_alias = "jobs")]
```

Comment:
```suggestion
    #[arg(long, short = 'j', visible_alias = "jobs")]
```

Reply:
yeah, that conflicts with short `--json` so opted for short `-t`

Reply:
I see, I would rather have jobs on j than json because it's more common

Reply:
:+1:  moved in https://github.com/foundry-rs/foundry/pull/8341/commits/b060828f47a31f54054522ef23f7f8df8c51bc2f